packages/core/src/api/stageInstances.ts

```diff
@@ -0,0 +1,75 @@
+/* eslint-disable jsdoc/check-param-names */
+
+import { type RequestData, type REST } from '@discordjs/rest';
+import {
+	type RESTGetAPIStageInstanceResult,
+	type RESTPatchAPIStageInstanceJSONBody,
+	type RESTPatchAPIStageInstanceResult,
+	type RESTPostAPIStageInstanceJSONBody,
+	type RESTPostAPIStageInstanceResult,
+	Routes,
+} from 'discord-api-types/v10';
+
+export class StageInstancesAPI {
+	public constructor(private readonly rest: REST) {}
+
+	/**
+	 * Creates a new stage instance
+	 *
+	 * @see {@link https://discord.com/developers/docs/resources/stage-instance#get-stage-instance}
+	 * @param body - The data to use when creating a new stage instance
+	 * @param options - The options to use when creating a new stage instance
+	 */
+	public async create(
+		body: RESTPostAPIStageInstanceJSONBody,
+		{ reason, signal }: Pick<RequestData, 'reason' | 'signal'> = {},
+	) {
+		return this.rest.post(Routes.stageInstances(), {
+			body,
+			reason,
+			signal,
+		}) as Promise<RESTPostAPIStageInstanceResult>;
+	}
+
+	/**
+	 * Gets a stage instance
```

Comment:
Other files (for example `GuildsAPI`) uses "fetches" instead of "gets".
```suggestion
	 * Fetches a stage instance
```